### ISSUE TITLE
Config Hotreload?

### ISSUE THREAD

**Issue by helixoid** [mierak/rmpc#168]
### Description

I am trying to get matugen to work with rmpc but I have to restart rmpc whenever I change the color scheme.
Would really like a feature like alacritty or other terminal with hotreload config.

**Comment by mierak** [comment]
Yep, its a nice to have feature, it is on my todo list, but not high priority atm.

**Comment by mierak** [comment]
Fixed in https://github.com/mierak/rmpc/pull/262 (and in https://github.com/mierak/rmpc/pull/261).

Seems to be working well, but I would be very surprised if there are no edge cases which needs fixing, please open up a new issue if you encounter any. https://mierak.github.io/rmpc/next/configuration/#enable_config_hot_reload
